next.config.js

```diff
@@ -131,7 +131,9 @@ const SentryWebpackPluginOptions = {
   disableClientWebpackPlugin: false,
 };
 
-module.exports = withSentryConfig(
-  withTM({ ...nextConfig }),
-  SentryWebpackPluginOptions
-);
+module.exports = withTM({ ...nextConfig });
```

Comment:
Temporarily commenting this out. We moved projects from Q5 account and they seem to not have been moved successfully for some reason. Commenting this out until its figured out